CMakeLists.txt

```diff
@@ -20,14 +20,16 @@ if(CGLM_USE_C99)
   set(C_STANDARD 99)
 endif()
 
-if("${CMAKE_C_COMPILER_ID}" STREQUAL "MSVC")
-  list(APPEND LDFLAGS "/W4")
-elseif("${CMAKE_C_COMPILER_ID}" STREQUAL "GNU")
-  set(C_EXTENSIONS YES)
-  list(APPEND LDFLAGS "-Wall" "-Wpedantic" "-Wstrict-aliasing=2")
-  list(APPEND LDFLAGS "-Werror=strict-prototypes" "-fstrict-aliasing" "-O3")
-elseif("${CMAKE_C_COMPILER_ID}" STREQUAL "CLANG")
-  list(APPEND LDFLAGS "-Wall" "-Weverything" "-Wpedantic")
+if(MSVC)
+	add_definitions(-DNDEBUG -D_WINDOWS -D_USRDLL -DCGLM_EXPORTS -DCGLM_DLL)
+	add_compile_options("/W3" "/Ox" "/Gy" "/Oi" "/TC" "/analyze")
+else()
+    add_compile_options("-Wall" "-Werror" "-O3")
+    if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
+		add_compile_options("-stdlib=libc++" "-Weverything")
```

Comment:
> -stdlib=libc++

May I ask why do we need to add this?

Reply:
My bad! I didn't copy the arguments correctly! Will fix!